transport/src/androidTest/kotlin/com/genesys/cloud/messenger/transport/core/messagingclient/MCMessageTests.kt

```diff
@@ -65,6 +71,9 @@ class MCMessageTests : BaseMessagingClientTest() {
             connectSequence()
             mockMessageStore.onMessageError(ErrorCode.MessageTooLong, "message too long")
         }
+        verify(exactly = 0) {
+            mockCustomAttributesStore.onError()
```

Comment:
should you add a verification for the CA state? it should be pending or sending?

Reply:
CustomAttributesStore is not a subject for this test and is mocked. So it is not going to get the proper state transition unless I explicitly implement mock behaviour for it. Something like:

every { mockCustomAttributesStore.onError() } answers mockCustomAttributes.state = State.Error. 

Or instead of mocking, I will need to provide an actual instance of CustomAttributesStore and then test the values. 

But Since we have a dedicated CustomAttributesStoreTest that knows how to test this state transitions, I have no need to do it here.